packages/editor/src/components/post-title/index.native.js

```diff
@@ -118,11 +119,20 @@ export default compose(
 			isPostTitleSelected,
 		} = select( 'core/editor' );
 
-		const { getSelectedBlockClientId } = select( 'core/block-editor' );
+		const { getSelectedBlockClientId, getBlockRootClientId, getLowestCommonAncestorWithSelectedBlock } = select( 'core/block-editor' );
+
+		const clientId = getSelectedBlockClientId();
+		const isAnyBlockSelected = !! clientId;
+		const parentId = getBlockRootClientId( clientId );
+		const firstToSelect = getLowestCommonAncestorWithSelectedBlock( clientId );
+		const isInnerBlock = parentId && firstToSelect !== parentId;
+
+		const isDimmed = isAnyBlockSelected && isInnerBlock;
```

Comment:
I think for the case of `PostTitle` this should be simpler? 

```js
const selectedId = getSelectedBlockClientId();
const selectionIsNested = !! getBlockRootClientId( selectedId ); // should return null when we pass undefined
```

And `selectionIsNested` should be enough to know when to dim the title.

Reply:
Thanks, done !